apiserver/plane/app/views/issue/base.py

```
@@ -46,6 +47,10 @@
     CycleIssue,
     UserRecentVisit,
     ModuleIssue,
+    IssueRelation,
+    User,
```

Ruff (0.11.9)

`plane.db.models.User` imported but unused

Remove unused import: `plane.db.models.User`

(F401)